### DISCUSSION TITLE
Will the performance improve if I use the NVIDIA A100 with 80GB?

### DISCUSSION THREAD

**Discussion by lsh7161** [hacksider/Deep-Live-Cam#467] [General]
Hello, as mentioned, I am using an RTX 3090, but when working with high-quality settings, I experience significant frame drops. I'm wondering if upgrading to the NVIDIA A100 with 80GB would prevent these frame drops. I'm really curious about this

**Comment by hacksider** [comment]
Are you using the live mode? As of now, we're still optimizing the performance. And usually it can only provide you half the fps of the webcam you're using since it's doing a post processing. Even if you upgrade to a higher GPU we can't guarantee a faster performance now. I always suggest not using the enhancer on live mode as of this moment. You can improve it though by using sharpen on OBS as well as nvidia super resolution filter.
